components/ui/base/proxy/nginx.conf

```diff
@@ -58,6 +58,11 @@ http {
             proxy_read_timeout 30m;
         }
 
+        location /apis/workspaces.konflux-ci.dev/ {
```

Comment:
shouldn't it be under `/api/k8s/`?

```suggestion
        location /api/k8s/apis/workspaces.konflux-ci.dev/ {
```

Reply:
yes, you're right, we should align with how the other clusters are configured.

Reply:
Had to do some path rewriting in nginx, but I've fixed it in a953f202.